src/plugins/intel_gpu/src/plugin/transformations/group_norm_composition.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "group_norm_composition.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/mvn.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/group_normalization.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/pass/pattern/op/or.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+namespace ov {
+namespace intel_gpu {
+
+GroupNormComposition::GroupNormComposition() {
```

Comment:
1. That can be a common pass.
2. Are new IRs generated w/ decomposed GroupNorm?
3. Rename to `GroupNormFusion` to be aligned with other passes

Reply:
1. Okay, I'll do it in a separate PR.
2. Yes, new IRs have decomposed GroupNorm layers.
3. Okay, I'll also apply it with #1.